plaso/storage/identifiers.py

```diff
@@ -103,3 +105,33 @@ def CopyToString(self):
       return '{0:s}.{1:d}'.format(self.name, self.row_identifier)
 
     return None
+
+class RedisKeyIdentifier(containers_interface.AttributeContainerIdentifier):
```

Comment:
I opt to rename to `RedisKeyContainerIdentifier` to make it clear its a container identifier type, not some other type of identifier

Reply:
good point, let's consider renaming all of them after this PR